tests/unit_tests/cuda/telescope_propagation_cuda.cpp

```diff
@@ -0,0 +1,144 @@
+/** Detray library, part of the ACTS project (R&D line)
+ *
+ * (c) 2022 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+// Project include(s).
+#include "telescope_propagation_cuda_kernel.hpp"
+
+// VecMem include(s).
+#include <vecmem/memory/cuda/managed_memory_resource.hpp>
+#include <vecmem/utils/cuda/copy.hpp>
+
+// GTest include(s).
+#include <gtest/gtest.h>
+
+TEST(propagation, telescope_geometry) {
+    /*
```

Comment:
I will comment out this test as telescope geometry doesn't take covfie field yet, and I am not sure of how to make it work right now..